src/views/Home.vue

```diff
@@ -0,0 +1,9 @@
+<script setup lang="ts">
+// This starter template is using Vue 3 <script setup> SFCs
+// Check out https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup
+import HomeLayout from "../layouts/HomeLayout.vue"
+</script>
+
+<template>
+  <HomeLayout/>
```

Comment:
Once you change the layout to do <slot></slot>, you don't really have to change this. But in the future there will be content added between the home layout tags.